framework/validators/DefaultValueValidator.php

```diff
@@ -37,14 +37,19 @@ class DefaultValueValidator extends Validator
      */
     public $skipOnEmpty = false;
 
+    /**
+     * @var bool this property forces the use of $value as a callback method to get the default value.
```

Comment:
`callableValue` is more readable
`@var bool whether the [[value]] is callback.`

Reply:
tks @WinterSilence

Reply:
@eecjimmy sorry, I'm miss `@since 2.0.46` + need update [CHANGELOG.md](https://github.com/eecjimmy/yii2/blob/master/framework/CHANGELOG.md)

Reply:
ok, got it